mappings/net/minecraft/resource/JsonResourceLoader.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS xg net/minecraft/resource/JsonResourceLoader
```

Comment:
I'd prefer `JsonDataLoader` as this is only used by data.

Reply:
Yeah, I wanted to somehow squeeze `Data` in there, but most classes in that package have `Resource` somewhere in there name (which makes sense, in that context), but I agree that `Data` is better.
It also seems to be only intended for data, as all the strings in that class all talk about data.

Reply:
It's fine IMO